examples/react-agent-durable/README.md

```
@@ -0,0 +1,123 @@
+# Terminal 4 - run the agent
+cd examples/react-agent-durable
+python main.py
+```
+
+`PYTHONPATH=.` puts this directory on the worker's import path so it can import
+`weather_agent`; run the worker from this directory.
+
+Expected output (Terminal 4): the model's final answer, followed by the tool calls
+it made durably, for example:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)